docs/runbooks/NoReadyVirtController.md

```diff
@@ -0,0 +1,35 @@
+# NoReadyVirtController
+
+## Meaning
+
+From a high-level perspective the virt-controller has all the cluster wide virtualization functionality.
```

Comment:
Would change `virt-controller has all the cluster wide virtualization functionality` to `virt-controller is responsible for all the cluster wide virtualization functionality`.

Reply:
@iholder-redhat Updated. Please review.